tools/cli/commands.go

```diff
@@ -180,8 +180,12 @@ const (
 	showErrorStackEnv    = `CADENCE_CLI_SHOW_STACKS`
 )
 
+func SetFactory(factory ClientFactory) {
+	CFactory = factory
+}
+
 var (
-	cFactory ClientFactory
+	CFactory ClientFactory
```

Comment:
probably change back to module private if you decided to use Set method

Reply:
Oh yes you are right. Thank you.